Read Author

justin-obrien-smoothstack opened the pull request:
Implemented `AdminAuthorController.readAuthor`. Manually ran some tests and documented the results.

justin-obrien-smoothstack committed:
Write readAuthor method in AdminAuthorController class

Don't test or implement content negotiation yet

justin-obrien-smoothstack committed:
Remove an unneeded blank line in AdminAuthorController.readAuthor

justin-obrien-smoothstack committed:
Merge branch 'dev-jo' into dev-jo-read-author

justin-obrien-smoothstack committed:
Merge branch 'dev-jo' into dev-jo-read-author

justin-obrien-smoothstack committed:
Add file to document AdminAuthorController tests

justin-obrien-smoothstack committed:
Fix RequestMapping path in AdminAuthorController.readAuthor

justin-obrien-smoothstack committed:
Update AdminAuthorController tests file

justin-obrien-smoothstack committed:
Change package for AdminAuthorController

justin-obrien-smoothstack committed:
Move package from src/tests to src/main

justin-obrien-smoothstack committed:
Put leading / on URI in @RequestMapping

justin-obrien-smoothstack committed:
Define a variable for the HTTP status in a method

AdminAuthorController.readAuthor
This makes the code a little cleaner and makes the return statements
more concise

justin-obrien-smoothstack committed:
Do AdminAuthorController tests & document results

justin-obrien-smoothstack committed:
Merge remote-tracking branch 'origin/dev' into dev-jo-read-author

trevorhuis-smoothstack reviewed (approved):
Thank you for cleaning up the author service

justin-obrien-smoothstack committed:
Untrack application.properties (again?)

justin-obrien-smoothstack committed:
Restore application.properties

justin-obrien-smoothstack committed:
Re-ignore application.properties, but don't untrack it

Hopefully this has the desired effect of not committing
application.properties, but allowing it to hang around in my local
working tree

trevorhuis-smoothstack reviewed (approved):
Looks good, very clean and i stole some concepts you used if you dont mind!

justin-obrien-smoothstack committed:
Auto-generate doc comments for AdminAuthorService.readAuthor

sean-darsie reviewed (approved):
Looks good. I like the way you did your error handling.

justin-obrien-smoothstack committed:
Merge remote-tracking branch 'origin/dev' into dev-jo-read-author
